File: Foundation/Basics/benjaminbulb.cpp
#include<iostream>
using namespace std;

void benjbulb(int n)
{
    for(int i= 1; i * i < n; i++)
    {
        cout<<i*i<<endl;

    }
}
int main()
{
    int n;
    cin>>n;
    benjbulb(n);
    return 0;
}

File: Foundation/Basics/benjaminbulb.java
import java.util.Scanner;
public class benjaminbulb
{
    public static Scanner scn = new Scanner(System.in);
    public static void benjbulb(int n)
    {
        for(int i = 1; i*i < n; i++)
        {
            System.out.println(i*i);
        }
    }
    public static void main(String[] args)
    {
        int n = scn.nextInt();
        benjbulb(n);
    }
}